<?xml version="1.0"?>
<launch>
	<arg name="TestDevice" value="Spacenav"/>	
	<node pkg="masterslave" type="MasterSlaveManipulation" name="MasterSlaveManipulation" args="$(arg TestDevice)" output="screen"/>
	<node pkg="masterslave" type="NumericKinematic" name="NumericKinematic" output="screen"/>
	<node pkg="masterslave" type="NumericKinematicCommander" name="NumericKinematicCommander" args="$(arg TestDevice)" output="screen"/>
	<node pkg="masterslave" type="TrajectoryGenerator" name="TrajectoryGenerator" output="screen"/>
</launch>
